src/rules/converters/comment-format.ts

```diff
@@ -0,0 +1,56 @@
+import { RuleConverter } from "../converter";
+
+export const CheckTrailingLowercaseMessage: string =
+    "Only first trailing comment can be configured.";
+export const CapitalizedIgnoreMessage: string =
+    "Only accepts a single string pattern to be ignored.";
+
+export const convertCommentFormat: RuleConverter = tslintRule => {
+    const capitalizedRuleArguments: string[] = [];
+    const spaceCommentRuleArguments: string[] = [];
+    const capitalizedNotices: string[] = [];
+
+    const hasCheckSpace = tslintRule.ruleArguments.includes("check-space");
+    const hasCheckLowercase = tslintRule.ruleArguments.includes("check-lowercase");
+    const hasCheckUppercase = tslintRule.ruleArguments.includes("check-uppercase");
+    const hasCheckTrailingLowercase = tslintRule.ruleArguments.includes("allow-trailing-lowercase");
+
+    if (!hasCheckSpace) {
+        spaceCommentRuleArguments.push("never");
+    }
+
+    if (hasCheckUppercase) {
+        capitalizedRuleArguments.push("always");
+    } else if (hasCheckLowercase) {
+        capitalizedRuleArguments.push("never");
+    }
+
+    if (hasCheckTrailingLowercase) {
```

Comment:
Doesn't ESLint's `ignoreConsecutiveComments` satisfy this use case?

Reply:
You are right! Didn't noticed about it earlier.

Reply:
I'm interpreting them to have the same intent: that only the first in a series of comments will be linted. ESLint's wording is a little confusing here.

Reply:
@JoshuaKGoldberg it is done, I removed it.